Controllers/getController.js

```diff
@@ -15,10 +15,18 @@ exports.getCategories = (req, res) => {
       next(err);
     });
 };
-exports.getReviews = (req, res) => {
-  selectReviews().then((reviews) => {
-    res.send({ reviews });
-  });
+exports.getReviews = (req, res, next) => {
+  const { category, sort_by, order } = req.query;
+
+  selectReviews(category, sort_by, order)
+    .then((reviews) => {
+      if (!reviews.length) {
+        res.send(reviews);
```

Comment:
You would still return the empty array on a key of reviews here, so no need for the if/else logic